osu.Game/Beatmaps/BeatmapSetOnlineInfo.cs

```diff
@@ -70,6 +70,16 @@ public class BeatmapSetOnlineInfo
         /// The availability of this beatmap set.
         /// </summary>
         public BeatmapSetOnlineAvailability Availability { get; set; }
+
+        /// <summary>
+        /// Beatmap set's song genre.
+        /// </summary>
+        public BeatmapSetOnlineGenre Genre { get; set; }
+
+        /// <summary>
+        /// Beatmap set's song language.
+        /// </summary>
+        public BeatmapSetOnlineLanguage Language { get; set; }
     }
 
     public class BeatmapSetOnlineCovers
```

Comment:
now that there are more enums and classes for `BeatmapSetOnlineInfo`, I suggest you just split every single one of them into their own files

Reply:
I've checked other cases and I'll instead combine them